pyproject.toml

```diff
@@ -83,5 +83,6 @@ module = [
     "pytorch_lightning.utilities.parsing",
     "pytorch_lightning.utilities.seed",
     "pytorch_lightning.utilities.xla_device",
+    "pytorch_lightning.utilities.parameter_tying",
```

Comment:
nice!

Reply:
super nit: let's keep the list alphabetized